storage/badger/operation/spork.go

```diff
@@ -29,16 +29,3 @@ func InsertSporkRootBlockHeight(height uint64) func(*badger.Txn) error {
 func RetrieveSporkRootBlockHeight(height *uint64) func(*badger.Txn) error {
 	return retrieve(makePrefix(codeSporkRootBlockHeight), height)
 }
-
-// InsertProtocolVersion inserts the protocol version for the present spork.
-// A single database and protocol state instance spans at most one spork, and
-// a spork has exactly one protocol version for its duration, so this is
-// inserted exactly once, when bootstrapping the state.
-func InsertProtocolVersion(version uint) func(*badger.Txn) error {
-	return insert(makePrefix(codeProtocolVersion), version)
```

Comment:
I feel we should also Deprecate the corresponding storage code:
https://github.com/onflow/flow-go/blob/0ba9fce316ff9c29d96e20dc2faa597b1cf2cf74/storage/badger/operation/prefix.go#L22